src/components/ha-circular-progress.ts

```diff
@@ -1,54 +1,39 @@
-import { CircularProgress } from "@material/mwc-circular-progress";
-import { CSSResultGroup, css } from "lit";
+import "element-internals-polyfill";
+import { MdCircularProgress } from "@material/web/progress/circular-progress";
+import { CSSResult, css } from "lit";
 import { customElement, property } from "lit/decorators";
 
 @customElement("ha-circular-progress")
-// @ts-ignore
-export class HaCircularProgress extends CircularProgress {
-  @property({ type: Boolean })
-  public active = false;
+export class HaCircularProgress extends MdCircularProgress {
+  @property({ attribute: "aria-label", type: String }) public ariaLabel =
+    "Loading";
```

Comment:
We should localize this

Reply:
Would it be accepted if I throw hass in the HaCircularProgress because I can also update every ha-circular-progress to have loading set?

Reply:
I would give it an optional `localize` property, if it isn't set, we fallback to "Loading"

Reply:
Use:

```
  @state()
  @consume({ context: localizeContext, subscribe: true })
  _localize!: LocalizeFunc;
  
  ${this._localize("ui.component.circular-progress.loading")}
```

Reply:
Is there anything special that needs to be done for the localizeContext to be available in demo pages? I can't seem to get it to work and other demo pages that have this like the entity button card is broken.